tfrecorder/converter.py

```diff
@@ -22,7 +22,7 @@
 
 import logging
 import os
-from typing import Any, Dict, Optional, Sequence, Tuple, Union
+from typing import Any, Dict, Optional, Sequence, Union
```

Comment:
I think internally I've seen us put each of these on a single line.  

eg 
from typing import Any
from typing import Dict
etc...

Do you think we should move towards that format here?

Reply:
Normally, we should be importing up to the module level only, but there's an explicit exception for the `typing` module:
https://google.github.io/styleguide/pyguide.html#typing-imports